config/sync/views.view.featured_recipe.yml

```diff
@@ -28,7 +28,7 @@ display:
         options:
           perm: 'access content'
       cache:
-        type: tag
+        type: none
```

Comment:
Why none on the cache type?

Reply:
The featured recipes don't display in random order on each visit unless the caching is turned off. When it's set to tag, the specific featured recipe displayed is cached and will continue to be displayed on every visit.

Reply:
Alright sound.